package.json

```diff
@@ -6,9 +6,13 @@
     "build": "next build && next export -o build",
     "start": "NODE_ENV=production next start"
   },
+  "devDependencies": {
+    "@prefresh/next": "^0.2.0",
```

Comment:
I tried with `@prefresh/next@0.3.0` and I got an error:

![image](https://user-images.githubusercontent.com/13313058/83325522-1f63f180-a26d-11ea-9ea4-70b07ad78048.png)

However, downgrading to `0.2.0` is working well. Do you know why that would happen?

Reply:
The import maps must be wrong for your node version, which one are you using?

Reply:
I'm using node 14.3.0

Reply:
`@prefresh/core` is being imported as commonjs, but the main export (`"."`) is only defined as supporting ESM: https://github.com/JoviDeCroock/prefresh/blob/51c583b9f2e830ac825e024428213821be1f82b2/packages/core/package.json#L7

Adding a `"default"` export would fix this.

Reply:
I can confirm that this was the problem! Thanks @developit @JoviDeCroock

Reply:
@aralroca

Did you add `default` to the export map?

Reply:
Yes, I touched directly in the `node_modules` on my project, and it works pretty well:

```diff
"exports": {
    ".": {
      "import": "./src/index.js",
+      "default": "./src/index.js"
    },
    "./package.json": "./package.json",
    "./": "./"
  },
```

I forked now the prefesh repo to do the PR, but you got faster 😊